andela/bp-esa-backend
# moved template  to root dir and mounted

#### What does this PR do?
- fixes bug with email template not found
- change email template dir
- copy template dir to docker image

#### Description of Task to be completed?

Creates a template directory in root and moved email templates to this directory. Adds them to the docker container on image build for deployment

#### How should this be manually tested?
1. On terminal, run command do `yarn start`
2. watch the logs on the terminal
3. when an automation kicks off (when a `GET` call is made to `mockapi/placements`) you should not see error due to email template not available

#### What are the relevant pivotal tracker stories?
#165197465
